helpers/toolset.sh

```diff
@@ -30,6 +30,14 @@ finalise_installation() {
     rm -f .partial
 }
 
+# Checks how many cores are available
+#
+# Returns:
+#     Number of available cores
+fiqure_out_processor_cores() {
+    echo $(nproc --all)
```

Comment:
The solution requires the program `/usr/bin/nproc` which has to be installed beforehand. We need a simple solution.
Use the following:
```
grep -c ^processor /proc/cpuinfo 
```